lib/named-locks.js

```diff
@@ -1,13 +1,13 @@
 // @ts-check
 const util = require('util');
 
-const sqldb = require('../prairielib/lib/sql-db');
+const { PostgresPool } = require('../prairielib/lib/sql-db');
 const sqlLoader = require('../prairielib/lib/sql-loader');
 
 const sql = sqlLoader.loadSqlEquiv(__filename);
 
 /** @typedef {import('pg').PoolClient} PoolClient */
-/** @typedef {{ client: PoolClient, done: (release?: any) => void }} Lock */
+/** @typedef {{ client: PoolClient }} Lock */
```

Comment:
Calling out that I got rid of an unused `done` on the Lock object.

Reply:
I think if anything, we should go the *other* direction and make it `{ __lockInternal }` or something - I think we really don't want people using this client to make queries of their own.

Reply:
I'm happy to just yell at anyone who tries 😄. To really hide it I think we could do:
```
const my_secret_symbol = Symbol();
const lock = { my_secret_symbol: client };
return lock;
```